spec/lib/world_taxonomy_sorter_spec.rb

```diff
@@ -1,12 +1,12 @@
-RSpec.describe WorldTaxonomySorter do
-  class FakeTaxon
-    attr_reader :title
+class FakeTaxon
+  attr_reader :title
 
-    def initialize(title)
-      @title = title
-    end
+  def initialize(title)
+    @title = title
   end
+end
```

Comment:
You probably don't need this to be a class in the global scope.

You can likely do:

```
let(:fake_taxon_class) do
  Class.new do
    attr_reader :title

    def initialize(title)
      @title = title
    end
  end
end
```

though I imagine this could be shorter still as `let(:fake_taxon_class) { Struct.new(:title) }` or even just replace the class with `OpenStruct.new(title: "title")` in the usage.